app/models/activity.rb

```diff
@@ -47,6 +47,7 @@ class Activity < ApplicationRecord
     :sector_step,
     :call_present_step,
     :call_dates_step,
+    :dates_step,
```

Comment:
I can't find where this lives!

Reply:
Not sure what you mean - if you mean looking for a definition like `def dates_step`, there isn't one. This is not a method, it's a symbol that gets its meaning from being used in the model validations:
```
validates :planned_start_date, presence: {message: I18n.t("activerecord.errors.models.activity.attributes.dates")}, on: :dates_step, unless: proc { |a| a.actual_start_date.present? }
```
and then being checked against in the validation context:
```
activity.valid?(:dates_step)
```